server/src-rsr/initialise.sql

```diff
@@ -607,3 +596,35 @@ CREATE TABLE hdb_catalog.hdb_allowlist
   collection_name TEXT UNIQUE
     REFERENCES hdb_catalog.hdb_query_collection(collection_name)
 );
+
+CREATE TABLE hdb_catalog.remote_schema_permissions
```

Comment:
Our naming convention for catalog tables is singular. Change it to `remote_schema_permission`?

Reply:
Except we already have `hdb_catalog.remote_schemas` but sure.